test/StakeManager.t.sol

```diff
@@ -108,19 +108,19 @@ contract StakeTest is StakeManagerTest {
         uint256 stakeAmount = 100;
         StakeVault userVault = _createStakingAccount(testUser, stakeAmount, 0, stakeAmount * 10);
 
-        (,, uint256 totalMP,,,,) = stakeManager.accounts(address(userVault));
+        (,, uint256 totalMP,,,,,) = stakeManager.accounts(address(userVault));
         assertEq(stakeManager.totalSupplyMP(), stakeAmount, "total multiplier point supply");
         assertEq(totalMP, stakeAmount, "user multiplier points");
 
         vm.prank(testUser);
         userVault.unstake(stakeAmount);
 
-        (,,, totalMP,,,) = stakeManager.accounts(address(userVault));
+        (,,, totalMP,,,,) = stakeManager.accounts(address(userVault));
         assertEq(stakeManager.totalSupplyMP(), 0, "totalSupplyMP burned after unstaking");
         assertEq(totalMP, 0, "userMP burned after unstaking");
     }
 
-    function test_restakeOnLocked() public {
+    function _test_restakeOnLocked() public {
```

Comment:
I wonder if we should remove all the tests related to restaking altogether.

There's also #103 , maybe we make it as part of that?

Reply:
I guess so, or refactor onto not tests.